### DISCUSSION TITLE
"Failed to find the end of the bit field: End of the message reached"

### DISCUSSION THREAD

**Discussion by DarkRider400** [FakeFishGames/Barotrauma#13766] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Simply hosting a server and waiting a few minutes, I get booted from _my own server_ without mods and with nobody else in the server. I have attached screenshot of the console log that was produced when I the server booted me.

![Screenshot (1)](https://github.com/FakeFishGames/Barotrauma/assets/32965100/0f193429-724d-403a-a24d-a97a5072d2a1)

I have also found a reddit thread describing identical errors encountered: https://www.reddit.com/r/Barotrauma/comments/1c131oa/help_keeps_crashing_while_playing_with_friend/

### Reproduction steps

1. Started a multiplayer server using the ingame menu.
2. Played for a few minutes.
3. Got booted from my own server. Had no mods running and no other players joined.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
It seems someone has discovered an exploit that allows crashing servers by sending them a specifically crafted malformed packet, and is abusing this to take down servers. We know where the issue is and are working on a fix. We'll hotfix this as soon as we can. Sorry for the inconvenience! :/

**Comment by Regalis11** [comment]
The issue should be fixed in the latest patch (v1.3.0.3). We have gotten a handful of reports of crashes similar to this after the hotfix though - it's possible there is still some kind of an issue left that the hackers are able to exploit. We are not completely sure of this though, the few reports we've gotten have been a little sparse on details and it could be the servers had not updated to the latest version. We'll keep the ticket open for a while longer while we monitor the situation!
